search-app/src/cmr/search/services/query_execution.clj

```diff
@@ -140,16 +140,69 @@
   [context query]
   [context (related-item-resolver/resolve-related-item-conditions query context)])
 
+(defn- get-facets-with-count
+  "Extract out the facet part that contains title and count, amoung other things: 
+  [{:title \"t1\" :count 0} {:title \"NonExist\" :count 0} {:title \"t3\" :count 1}]
+  from the facets result."
```

Comment:
Maybe add something in the docstring about why you're taking the first child of `facets-result`